e2e/app/page/access-renewal-mode.ts

```diff
@@ -22,7 +21,8 @@ export default class AccessRenewalPage extends AuthenticatedPage {
   }
 
   async hasExpired(): Promise<boolean> {
-    return await this.containsText(LabelAlias.AccessExpired);
+    const notificationText = await getNotificationText(page, DataTestIds.AccessRenewal);
```

Comment:
having trouble here... can you help @aweng98 ?

it's timing out looking for an element.  I want it to give me a "false" instead of a timeout when it doesn't find that element.

Reply:
OK I think I have it figured out now... see StaticText.assertNotPresent() / assertNoNotification() and AccessRenewalMode.hasNotExpired()